ci/install.sh

```diff
@@ -11,13 +11,13 @@ main() {
         target=x86_64-apple-darwin
     fi
 
-    # TODO At some point you'll probably want to use a newer release of `cross`,
-    # simply change the argument to `--tag`.
+    local tag="$(git ls-remote --tags --refs --exit-code https://github.com/japaric/cross | cut -d/ -f3 | sort --version-sort | tail -n1)"
```

Comment:
Let's also add a comment saying that this will install the latest 0.1.x release.